src/Command/User/CreateCommand.php

```diff
@@ -9,18 +9,24 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
+use Yiisoft\Rbac\Manager;
+use Yiisoft\Rbac\StorageInterface;
 use Yiisoft\Yii\Console\ExitCode;
 use Yiisoft\Yii\Cycle\Command\CycleDependencyProxy;
 
 class CreateCommand extends Command
```

Comment:
The command currently fails with:

![image](https://user-images.githubusercontent.com/47294/92290161-c9fa4900-ef1b-11ea-80e2-7b7651ec4790.png)

Reply:
A command to set RBAC role for existing user would be handy.

Reply:
> The command currently fails with:
> 
> ![image](https://user-images.githubusercontent.com/47294/92290161-c9fa4900-ef1b-11ea-80e2-7b7651ec4790.png)

I couldn't reproduce  error. Most likely you have old db schema